<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>媒体查看器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="file.qrc">
    <normaloff>:/root/app.png</normaloff>:/root/app.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QPushButton" name="openButton">
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QLabel" name="viewlabel">
      <property name="styleSheet">
       <string notr="true">background-color:white</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLineEdit" name="dirEdit"/>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QPushButton" name="stopButton">
      <property name="text">
       <string>暂停视频</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="playButton">
      <property name="text">
       <string>播放视频</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>339</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>控制(&amp;C)</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionStop"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuControl"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>播放(&amp;P)</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>暂停(&amp;S)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="file.qrc"/>
 </resources>
 <connections/>
</ui>
